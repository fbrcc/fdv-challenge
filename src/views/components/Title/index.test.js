import React from 'react';
import { Title } from './index'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<Title text="Ejercicio Intive"></Title>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });