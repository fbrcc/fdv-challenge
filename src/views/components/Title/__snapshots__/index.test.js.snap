// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa kmXoBq"
>
  <h2
    className="sc-bwzfXH furKEP"
  >
    Ejercicio Intive
  </h2>
</div>
`;
